ports/unix/coveragecpp.cpp

```diff
@@ -17,10 +17,57 @@ extern "C" {
 #include <py/runtime.h>
 }
 
+// Invoke all (except one, see below) public API macros which initialize structs to make sure
```

Comment:
Might be worth splitting this change to `coveragecpp.cpp` out into a separate commit?

Reply:
Yeah sure, done.